### ISSUE TITLE
feat(autocomplete): add support for aria-describedby

### ISSUE THREAD

**Issue by cai-feng** [angular/material#11004]
Adding `aria-describedby` to `md-autocomplete` like following code won't work.
```html
<form name="projectForm">
  <md-autocomplete md-input-id="input-project-name"
                  md-input-name="projectName"
                  md-no-asterisk
                  md-floating-label="Project name"
                  md-items="item in $ctrl.projects"
                  md-item-text="item.name"
                  md-selected-item="$ctrl.project"
                  md-select-on-match="true"
                  md-autofocus
                  required
                  aria-describedby="project-name-description"
                  flex>
  <md-item-template>
      <span>
        {{item.name}}
      </span>
  </md-item-template>
  <div ng-messages="projectForm.projectName.$error"
        ng-if="projectForm.projectName.$touched">
    <div ng-message="required">
      Project name is required.
    </div>
  </div>
  <div id="project-name-description">
    Descriptive text
  </div>
  </md-autocomplete>
</form>
```

**Comment by cai-feng** [comment]
@Splaktar Sorry for confusing and replying so late.

- This issue should impact many screen readers (ChromVox, VoiceOver and JAWS).
- Yes. Transferring `aria-describedby` from `md-autocomplete` to its `input` makes sense to its users.
- Keep you screen reader open and focus on the autocomplete by moving from other element.
  Ideally, you will hear something like this `{content of md-floating-label} Collapsed Required Combo box { content of aria-describedby}`

I also made a demo for you. thanks.
https://stackblitz.com/edit/angularjs-gx57rp

**Comment by Splaktar** [comment]
@cai-feng perfect, thank you. We should be able to get support for `aria-describedby` into 1.1.11.

If you are also interested in support for `aria-labelledby`, please make a comment on https://github.com/angular/material/issues/10815. I have that code ready to go, I just need to make sure that it is still a desired enhancement.

**Comment by Splaktar** [comment]
I opened PR https://github.com/angular/material/pull/11405 to address this.
